lib/diagnostics_channel.js

```diff
@@ -53,6 +61,8 @@ class ActiveChannel {
         const onMessage = this._subscribers[i];
         onMessage(data, this.name);
       } catch (err) {
+        if (err instanceof DCInterruptError) throw err;
```

Comment:
This exits the for loop and as a result other subscribers don't get the event.

Reply:
> Throwing an instance of `DCInterruptError` inside a subscriber will prevent subsequent subscriptions from running, and will forward the exception to the publisher

Although I think that it's a really good idea to stop sending messages if at least one of the messages is interrupted, I agree according to this implementation, only the first subscribed will receive this event, which seems incorrect. But from the security perspective, it seems like a good approach, since if a message was somehow interrupted, we might want to stop communication immediately, in case of potential security leaks, because that message may contain "poisoned" data.